spec/rubocop/cop/sorbet/rbi_versioning/valid_gem_version_annotations_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+RSpec.describe(RuboCop::Cop::Sorbet::ValidGemVersionAnnotations, :config) do
+  it "does not register an offense when comment is not a version annotation" do
+    expect_no_offenses(<<~RUBY)
+      # a random comment
+    RUBY
+  end
+
+  it "does not register an offense when comment is a valid version annotation" do
+    expect_no_offenses(<<~RUBY)
+      # @version = 1.3.4-prerelease
+    RUBY
+  end
+
+  it "does not register an offense when comment uses AND version annotations" do
+    expect_no_offenses(<<~RUBY)
+      # @version > 1, < 3.5
+    RUBY
+  end
+
+  it "does not register an offense when comment uses OR version annotations" do
+    expect_no_offenses(<<~RUBY)
+      # @version > 1.3.6
+      # @version <= 4
+    RUBY
+  end
+
+  it "registers an offense when gem version is not formatted correctly" do
```

Comment:
Let's add a test for an empty `@version` without an operator and one for `@version foo`.

Reply:
Done!